terminus-ui/checkbox/src/checkbox.component.scss

```diff
@@ -60,12 +60,27 @@ $ts-select-item-height: 3em !default;
   }
 
   // Target checkboxes inside a material menu dropdown panel
-  .mat-menu-panel & {
+  .ts-menu__panel & {
     // Match the material button line-height and spacing
     .c-checkbox {
       line-height: 2em;
       padding: 0 spacing(default);
     }
+
+    &.cdk-drag {
+      .c-checkbox {
+        padding: 0 spacing(large, 1) 2px spacing(default);
+      }
+    }
+
+    // By default the label doesn't fill the width even though the cursor signifies interaction
+    .mat-checkbox-layout {
+      width: 100%;
+    }
+
+    .mat-checkbox-label {
+      flex: 1;
+    }
```

Comment:
Do we overwrite material css here?

Reply:
Yep. Checkbox is still 95% material. For some reason they change the cursor to `pointer` for the entire label wrapper even though the actual clickable section is much smaller. This fixes that slightly confusing issue.